templates.go

```diff
@@ -6,7 +6,7 @@
 
 package main
 
-const validationStatesWithResultsTemplate = `{{ range . }}
+const validationStateWithResultsTemplate = `
 - ID: {{ .ID }}
```

Comment:
This is no longer an array. (i.e. remove intending with `- `)

Reply:
Un-indented the entire block.